components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-profile/reporting-profile.component.ts

```diff
@@ -126,14 +128,14 @@ export class ReportingProfileComponent implements OnInit, OnDestroy {
     this.openControls = {};
   }
 
-  getNodes(filters, params): Observable<Array<any>> {
-    const profileFilter = {type: { name: 'profile_id' } , value: { text: params.profileId}};
-    const controlFilter = {type: { name: 'control_id' } , value: { text: params.controlId}};
-    filters = [profileFilter, controlFilter].concat(filters);
```

Comment:
Here was the bug. The 'filters' variable is a 'ReportQuery' object and should not be concated with the profileFilter and controlFilter.